ailment/utils.py

```diff
@@ -12,7 +15,12 @@
     from .expression import Expression
 
 
-def get_bits(expr: Union[claripy.ast.Bits,'Expression',int]) -> Optional[int]:
+get_bits_type_params = Union[int, 'Expression']
+if claripy:
+    get_bits_type_params = Union[int, claripy.ast.Bits, 'Expression']
```

Comment:
\* expansion for subscript parameters isn't supported until Python 3.11, so I think until then this is the best I could come up with for handling dynamic type parameters, let me know if you think there is a better way.

Reply:
An alternative to this would be to guard the claripy import behind `if typing.TYPE_CHECKING`, but that would make it such that the dependency would exist for type checking, which seems imperfect.